src/Symfony/Component/DependencyInjection/Definition.php

```diff
@@ -265,15 +265,15 @@ public function addArgument($argument)
     public function replaceArgument($index, $argument)
     {
         if (0 === \count($this->arguments)) {
-            throw new OutOfBoundsException('Cannot replace arguments if none have been configured yet.');
+            throw new OutOfBoundsException(sprintf('Cannot replace arguments if none have been configured yet. Class: %s', $this->class));
```

Comment:
Let's try to add the class in the message itself like:

```suggestion
            throw new OutOfBoundsException(sprintf('Cannot replace arguments for class "%s" if none have been configured yet.', $this->class));
```

Can you try to do the same for the other ones as well?

Reply:
If changed all the messages to contain the class names properly